CHANGELOG.md

```diff
@@ -3,13 +3,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+
 ## [Unreleased]
 
 ## [1.0.0] - 2017-03-15
 ### Added
 - made metrics aware of multiple chips (@GhostLyrics)
 - made metrics also collect power draw (@GhostLyrics)
 - support for Ruby 2.3.0 (@eheydrick)
+- added temperature check (@holygits)
```

Comment:
Your change should be in Unreleased since 1.0.0 was already released without your PR a few days ago.